package.json

```diff
@@ -24,7 +24,7 @@
     "conventional-changelog-conventionalcommits": "^7.0.0",
     "execa": "^9.2.0",
     "rollup": "^4.18.0",
-    "turbo": "^2.0.4"
+    "turbo": "^2.2.3"
```

Comment:
Been following the guide at https://vercel.com/docs/monorepos/remote-caching#enable-and-disable-remote-caching-for-your-team.

**Step 1**

"Remote Caching" is enabled for the "next-intl" team on Vercel.

**Step 2**

Running `npx turbo login` locally yields this error:

```
❯ npx turbo login

turbo 2.2.3

  × Error making HTTP request: error decoding response body: missing field `origin` at line 1
  │ column 258
  ├─▶ error decoding response body: missing field `origin` at line 1 column 258
  ╰─▶ missing field `origin` at line 1 column 258
```

**Step 3**

Running `npx turbo link` did work though.

**Step 4**

(skipped)

**Step 5**

Locally Turbo caching works as expected (was already the case before).

---

**CI Config**
- `TURBO_TEAM` is set to `next-intl` in a GH Actions variable
- `TURBO_TOKEN` is set in a GH Actions secret

https://github.com/amannn/next-intl/blob/fd681ad84bde187ecaa828907c429c505f2d71cb/.github/workflows/main.yml#L11-L13

Reply:
Here's the most recent CI run: https://github.com/amannn/next-intl/actions/runs/11474295438/job/31929959155?pr=1449.

At the beginning of "Run pnpm run build" there are these entries:

![Screenshot 2024-10-23 at 08 29 32](https://github.com/user-attachments/assets/ff2c76fd-6a7f-447c-96f1-32e847ffbafe)